example-app/SantokuApp/src/features/demo-map/pages/DemoMapPage.tsx

```diff
@@ -0,0 +1,277 @@
+import {Button} from 'bases/ui/button/Button';
+import {TextInput} from 'bases/ui/input/TextInput';
+import {MapView, MapViewRef} from 'bases/ui/map/MapView';
+import {Marker, MarkerProps} from 'bases/ui/map/Marker';
+import {Spacer} from 'bases/ui/spacer/Spacer';
+import {SpecAndSourceCodeLink} from 'features/demo-github-link/components/SpecAndSourceCodeLink';
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import {View, StyleSheet, KeyboardAvoidingView, Platform} from 'react-native';
+import {Text} from 'react-native-elements';
+import {ScrollView} from 'react-native-gesture-handler';
+import {Region, MapType} from 'react-native-maps';
+
+import {MapTypePicker} from '../components/MapTypePicker';
+import {ToggleButton} from '../components/ToggleButton';
+import {initialMarker} from '../constants/initialMarker';
+import {initialRegion} from '../constants/initialRegion';
+import {RegionFormValues, useRegionForm} from '../form/RegionForm';
+
+export const DemoMapPage: React.FC = () => {
+  // TODO マーカーを追加するフォーム（緯度、経度、タイトル、説明、dragableを入力させる）
+
+  // 現時点でsetMarkerListを使っていないが、後でマーカーを画面から追加できるようにするためにマーカーの配列をstate管理する。
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [markerList, setMarkerList] = useState<MarkerProps[]>([initialMarker]);
+
+  const [region, setRegion] = useState<Region>(initialRegion);
+
+  const [mapType, setMapType] = useState<MapType>('standard');
+  const [showBuildings, setShowBuildings] = useState<boolean>(true);
+
+  const onSubmit = useCallback(
+    (values: RegionFormValues) =>
+      setRegion({
+        latitude: Number(values.latitude),
+        longitude: Number(values.longitude),
+        latitudeDelta: Number(values.latitudeDelta),
+        longitudeDelta: Number(values.longitudeDelta),
+      }),
+    [setRegion],
+  );
+  const {form} = useRegionForm({
+    initialValues: {
+      latitude: initialRegion.latitude.toString(),
+      longitude: initialRegion.longitude.toString(),
+      latitudeDelta: initialRegion.latitudeDelta.toString(),
+      longitudeDelta: initialRegion.longitudeDelta.toString(),
+    },
+    onSubmit,
+  });
+
+  const keyboardType = Platform.select({
+    ios: 'numbers-and-punctuation',
+    android: 'number-pad',
+  } as const);
+
+  const [scrollEnable, setScrollEnable] = useState<boolean>(true);
+  const [zoomEnabled, setZoomEnabled] = useState<boolean>(true);
+  const [rotateEnabled, setRotateEnabled] = useState<boolean>(true);
+  const [pitchEnabled, setPitchEnabled] = useState<boolean>(true);
+
+  const mapViewRef = useRef<MapViewRef>(null);
+  const animateToRegion = useCallback(() => mapViewRef.current?.animateToRegion(region), [region]);
+
+  useEffect(() => {
+    animateToRegion();
+  }, [animateToRegion, region]);
```

Comment:
```suggestion
  }, [animateToRegion]);
```

マーカーの実装で `const animateToRegion` が使われなければ、 `const animateToRegion` は `useCallback` 無しで `useEffect` の中に移動で良いと思います。

Reply:
`animateToRegion`内の処理を`useEffect`に移動しましたー（マーカーの実装では使用していません）
3501b87a57c76983a660c3dfc5e9a248ac0c9fef